diff --git a/node_modules/react-native-screens/lib/commonjs/components/ScreenStackHeaderConfig.js b/node_modules/react-native-screens/lib/commonjs/components/ScreenStackHeaderConfig.js
index e800f83..4da8c35 100644
--- a/node_modules/react-native-screens/lib/commonjs/components/ScreenStackHeaderConfig.js
+++ b/node_modules/react-native-screens/lib/commonjs/components/ScreenStackHeaderConfig.js
@@ -69,8 +69,7 @@ const ScreenStackHeaderConfig = exports.ScreenStackHeaderConfig = /*#__PURE__*/_
     onPressHeaderBarButtonItem: onPressHeaderBarButtonItem,
     onPressHeaderBarButtonMenuItem: onPressHeaderBarButtonMenuItem,
     ref: ref,
-    style: styles.headerConfig,
-    pointerEvents: "box-none",
+    style: [styles.headerConfig, styles.pointerEventsBoxNone],
     synchronousShadowStateUpdatesEnabled: _flags.default.experiment.synchronousHeaderConfigUpdatesEnabled
   }));
 });
@@ -146,6 +145,9 @@ const styles = _reactNative.StyleSheet.create({
     // We only want to center align the subviews on iOS.
     // See https://github.com/software-mansion/react-native-screens/pull/2456
     alignItems: _reactNative.Platform.OS === 'ios' ? 'center' : undefined
+  },
+  pointerEventsBoxNone: {
+    pointerEvents: 'box-none'
   }
 });
 //# sourceMappingURL=ScreenStackHeaderConfig.js.map
diff --git a/node_modules/react-native-screens/lib/module/components/ScreenStackHeaderConfig.js b/node_modules/react-native-screens/lib/module/components/ScreenStackHeaderConfig.js
index 71cd007..20677a0 100644
--- a/node_modules/react-native-screens/lib/module/components/ScreenStackHeaderConfig.js
+++ b/node_modules/react-native-screens/lib/module/components/ScreenStackHeaderConfig.js
@@ -65,8 +65,7 @@ export const ScreenStackHeaderConfig = /*#__PURE__*/React.forwardRef((props, ref
     onPressHeaderBarButtonItem: onPressHeaderBarButtonItem,
     onPressHeaderBarButtonMenuItem: onPressHeaderBarButtonMenuItem,
     ref: ref,
-    style: styles.headerConfig,
-    pointerEvents: "box-none",
+    style: [styles.headerConfig, styles.pointerEventsBoxNone],
     synchronousShadowStateUpdatesEnabled: featureFlags.experiment.synchronousHeaderConfigUpdatesEnabled
   }));
 });
@@ -137,6 +136,9 @@ const styles = StyleSheet.create({
     // We only want to center align the subviews on iOS.
     // See https://github.com/software-mansion/react-native-screens/pull/2456
     alignItems: Platform.OS === 'ios' ? 'center' : undefined
+  },
+  pointerEventsBoxNone: {
+    pointerEvents: 'box-none'
   }
 });
 //# sourceMappingURL=ScreenStackHeaderConfig.js.map
diff --git a/node_modules/react-native-screens/src/components/ScreenStackHeaderConfig.tsx b/node_modules/react-native-screens/src/components/ScreenStackHeaderConfig.tsx
index 6ca8ed7..fc8486c 100644
--- a/node_modules/react-native-screens/src/components/ScreenStackHeaderConfig.tsx
+++ b/node_modules/react-native-screens/src/components/ScreenStackHeaderConfig.tsx
@@ -118,8 +118,7 @@ export const ScreenStackHeaderConfig = React.forwardRef<
       onPressHeaderBarButtonItem={onPressHeaderBarButtonItem}
       onPressHeaderBarButtonMenuItem={onPressHeaderBarButtonMenuItem}
       ref={ref}
-      style={styles.headerConfig}
-      pointerEvents="box-none"
+      style={[styles.headerConfig, styles.pointerEventsBoxNone]}
       synchronousShadowStateUpdatesEnabled={
         featureFlags.experiment.synchronousHeaderConfigUpdatesEnabled
       }
@@ -227,4 +226,7 @@ const styles = StyleSheet.create({
     // See https://github.com/software-mansion/react-native-screens/pull/2456
     alignItems: Platform.OS === 'ios' ? 'center' : undefined,
   },
+  pointerEventsBoxNone: {
+    pointerEvents: 'box-none',
+  },
 });
